vm_scripts/machines.py

```
@@ -0,0 +1,692 @@
+from pyVmomi import vim
+from pyVim import connect
```

Ruff (0.8.2)

`pyVim.connect` imported but unused

Remove unused import: `pyVim.connect`

(F401)